xonsh/base_shell.py

```diff
@@ -118,10 +117,10 @@ def default(self, line):
         src, code = self.push(line)
         if code is None:
             return
-        hist = builtins.__xonsh_history__
+        hist = builtins.__xonsh_history__  # pylint: disable=no-member
         ts1 = None
-        tee = Tee() if builtins.__xonsh_env__.get('XONSH_STORE_STDOUT') \
-                    else io.StringIO()
+        store_stdout = builtins.__xonsh_env__.get('XONSH_STORE_STDOUT', False)  # pylint: disable=no-member
```

Comment:
Related to this my intent is to eventually replace references to builtins.__xonsh_env__ with something like builtinsx.xonsh.env. That is, rename built_ins.py to builtinsx.py. Then replace the global functions and symbols such as "ENV" with a class that encapsulates that functionality and is exposed as a singleton named builtinsx.xonsh. That will eliminate all of the references to xonsh specific double-underscore references of the builtins module. That will eliminate the need to suppress a few hundred "no-member" lint errors as well as be clearer and more concise.